net/net.go

```diff
@@ -165,11 +165,14 @@ func AddIPOffset(base *big.Int, offset int) net.IP {
 }
 
 // RangeSize returns the size of a range in valid addresses.
-func RangeSize(subnet *net.IPNet) int64 {
+func RangeSize(subnet *net.IPNet, boundary uint) int64 {
 	ones, bits := subnet.Mask.Size()
 	if bits == 32 && (bits-ones) >= 31 || bits == 128 && (bits-ones) >= 127 {
```

Comment:
my order of operations is rusty... was this line already supposed to protect against (bits-ones) being big enough to cause an overflow?

Reply:
or should we add a condition that returns 0 if (bits-ones) > 63?